### PR TITLE
Feat/pn 2280 dati statistici

### PR DESCRIPTION
**Pull request by LorenzoCocciarelli**
## Short description
This PR adds Statistics page with routing and heading for the page

## List of changes proposed in this pull request
- Adds Statistics page (Statistics.page.tsx) with title box
- Adds text render test for Statistics page (Statistics.page.test.tsx)
- Adds redux slice for statistics

## How to test
- Statistics page is visible in the sidebar
- Route brings to Statistics page correctly
- Title, subtitle and Export button are visible in Statistics page
- Reference from [Piattaforma Notifiche - Dashboard](https://www.figma.com/file/HjyZhnoAKbzCbxkmQCGsZw/Piattaforma-Notifiche?node-id=21749%3A137881) su Figma

### CODE REVIEW

File: packages/pn-pa-webapp/src/navigation/routes.tsx [hunk 1]
```diff
@@ -19,6 +20,7 @@ function Router() {
           <Route path={routes.DASHBOARD} element={<Dashboard />} />
           <Route path={routes.DETTAGLIO_NOTIFICA} element={<NotificationDetail />} />
           <Route path={routes.NUOVA_NOTIFICA} element={<NewNotification />} />
+          <Route path={routes.STATISTICHE} element={<Statistics />} />
```
Comment (AndreaCimini90) [inline packages/pn-pa-webapp/src/navigation/routes.tsx:23]: Anche se la voce di menù è nascosta, se l'utente ha ruolo operatore accede comunque alla pagina delle statistiche inserendo la url direttamente nella barra del browser.
come soluzione si potrebbe definire la rotta delle statistiche in questo modo
`<Route path={routes.STATISTICHE} element={<RouteGuard roles={[PNRole.ADMIN]} />}>
            <Route path={routes.STATISTICHE} element={<Statistics/>} />
          </Route>`
allo stesso livello di
`<Route path="/"  element={<RouteGuard roles={[PNRole.ADMIN, PNRole.OPERATOR]} />}>`

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

**Comment by AndreaCimini90** [comment]
Oltre ai commenti puntali, in fase di navigazione verso la pagina delle statistiche, compare il seguente errore
`Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
    at p
    at https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:24566:66
    at Typography (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:350014:87)
    at div
    at https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:24566:66
    at Grid (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:321586:87)
    at p
    at https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:24566:66
    at Typography (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:350014:87)
    at div
    at https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:24566:66
    at Grid (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:321586:87)
    at div
    at https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:24566:66
    at Grid (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:321586:87)
    at TitleBox (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:7932:5)
    at div
    at https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:24566:66
    at Box (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:361032:72)
    at Statistics (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:17306:68)
    at Outlet (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:488775:26)
    at RouteGuard (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:15331:5)
    at Outlet (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:488775:26)
    at SessionGuardRender (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:15461:85)
    at SessionGuard (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:15524:81)
    at Routes (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:488867:5)
    at Router
    at ErrorBoundary (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:3136:5)
    at main
    at https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:24566:66
    at Box (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:370907:72)
    at div
    at https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:24566:66
    at Stack (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:340196:87)
    at div
    at https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:24566:66
    at Stack (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:340196:87)
    at ErrorBoundary (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:3136:5)
    at Layout (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:4198:5)
    at App (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:14073:67)
    at Suspense
    at InnerThemeProvider (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:359758:70)
    at ThemeProvider (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:359279:5)
    at ThemeProvider (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:359778:5)
    at Router (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:488800:15)
    at BrowserRouter (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:488280:5)
    at Provider (https://portale-pa.dev.pn.pagopa.it/static/js/bundle.js:485723:20)
`

File: packages/pn-pa-webapp/src/pages/__test__/Statistics.page.test.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  useTranslation: () => ({
+    t: (str: string) => str,
+  }),
+}));
+
+describe('Statistics Page tests', () => {
+  const mockDispatchFn = jest.fn();
+
+  beforeEach(async () => {
+    const useDispatchSpy = jest.spyOn(redux, 'useDispatch');
```
[the remark this answers is not in the document]
Comment (LorenzoCocciarelli) [reply packages/pn-pa-webapp/src/pages/__test__/Statistics.page.test.tsx]: Credo di sì, rimosso

**Comment by LorenzoCocciarelli** [comment]
Corretto il DOM nesting per la titlebox

**Comment by LorenzoCocciarelli** [comment]
Corretto il test che falliva role.utility.test.ts

**Comment by LorenzoCocciarelli** [comment]
Aggiunta la route guard per la pagina Statistiche (Statistics.page.tsx)

**Pull request merged**